docroot/themes/humsci/su_humsci_theme/scss/base/_paragraphs.scss

```diff
@@ -0,0 +1,31 @@
+@charset "UTF-8";
+
+//
+// Paragraphs
+//
+
+// Decanter
+@import
+  'decanter-no-markup';
+
+// Default spacing
+.paragraph-item,
+.field-hs-page-hero {
```

Comment:
This isn't a paragraph. should it live in another file? like maybe `css/nodes/hs_page.scss`

Reply:
if you put it in an hs_page.scss we'll want a new library similar to the person

Reply:
Yes, yes, yes.